modules/gdscript/gdscript_parser.cpp

```diff
@@ -1033,10 +1046,18 @@ GDScriptParser::VariableNode *GDScriptParser::parse_property(VariableNode *p_var
 			}
 		}
 
+		while (match(GDScriptTokenizer::Token::ANNOTATION)) {
+			AnnotationNode *const new_annotation = parse_annotation(AnnotationInfo::TargetKind::FUNCTION);
```

Comment:
```suggestion
		while (match(GDScriptTokenizer::ANNOTATION)) {
			AnnotationNode *new_annotation = parse_annotation(AnnotationInfo::FUNCTION);
```

While this is not strictly regulated, in this codebase we generally do not use optional enum names and const pointers.